### DISCUSSION TITLE
Best way to insert custom graphic content

### DISCUSSION THREAD

**Discussion by RymMichaut** [Avaiga/taipy#136] [Q&A] [answered]
Hi everyone,
congrats for this wonderful framework, it’s a pleasure to use !
That beeing said, for my first app I need a gantt chart.
I think this type of chart is not supported yet by Taipy, hence I’d like to use another js lib.
Can I add a custom content to my webapp, and if so, what is the best way to do it ?

Thx in advance

**Comment by FlorianJacta** [comment, marked as answer]
Hi,

Taipy provides a way to insert Javascript libraries into Taipy. You can see more details [here](https://docs.taipy.io/en/latest/manuals/gui/extension/). This extension allows developers to bring in external graphical objects.

Gantt charts are now possible in Taipy. You will find documentation [here](https://docs.taipy.io/en/latest/manuals/gui/viselements/charts/gantt/) on creating your chart.

I hope it answers your questions.

**Comment by vhfgosselin** [comment]
Florian,

I remember that we create a simple Gantt chart in the past for Irv lustig.
It was just a pivoted bar chart. Do you remember?

Vincent
